{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -135,6 +138,17 @@ run_docker: populate_aws_id
             -e "AWS_SECRET_ACCESS_KEY=$(shell aws --profile default configure get aws_secret_access_key)" \
             --name=$(PROJECT_NAME) $(PROJECT_NAME):latest
 
+## Run Jupyter in a container and hit it in Chrome
+start_jupyter: populate_aws_id
+	docker run -dt -p 7777:8888 \
+            -e "AWS_ACCOUNT_ID=$(AWS_ACCOUNT_ID)" \
+            -e "AWS_ACCESS_KEY_ID=$(shell aws --profile default configure get aws_access_key_id)" \
+            -e "AWS_SECRET_ACCESS_KEY=$(shell aws --profile default configure get aws_secret_access_key)" \
+            --name=$(PROJECT_NAME) $(PROJECT_NAME):latest \
+	    jupyter notebook --ip 0.0.0.0 --allow-root \
+	&& sleep 1 \
+        && /usr/bin/open -a "/Applications/Google Chrome.app" 'http::/localhost:7777'
```

Comment:
Nice touch opening Chrome as part of the command. Couldn't possibly be more convenient.  With that, maybe even I will run my notebooks within the container!